src/commands/env/resume/sandbox.ts

```diff
@@ -0,0 +1,233 @@
+import {
+  GlobalInfo,
+  Lifecycle,
+  Messages,
+  Org,
+  ResultEvent,
+  SandboxEvents,
+  SandboxProcessObject,
+  ResumeSandboxRequest,
+  SandboxUserAuthResponse,
+} from '@salesforce/core';
+import { Duration } from '@salesforce/kit';
+import * as Interfaces from '@oclif/core/lib/interfaces';
+import { SandboxCommandBase } from '../../../shared/sandboxCommandBase';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-env', 'resume.sandbox');
+
+type FlagsDef = {
+  'set-default': boolean;
+  alias: string;
+  'poll-interval': Duration;
+  wait: Duration;
+  json: boolean;
+  name: string;
+  'job-id': string;
+  'target-org': Org;
+  'use-most-recent': boolean;
+};
+
+export default class ResumeSandbox extends SandboxCommandBase<SandboxProcessObject> {
+  public static summary = messages.getMessage('summary');
+  public static description = messages.getMessage('description');
+  public static examples = messages.getMessages('examples');
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public static flags: Interfaces.FlagInput<any> = {
+    'set-default': Flags.boolean({
+      char: 's',
+      summary: messages.getMessage('flags.setDefault.summary'),
+    }),
+    alias: Flags.string({
+      char: 'a',
+      summary: messages.getMessage('flags.alias.summary'),
+      description: messages.getMessage('flags.alias.description'),
+    }),
+    wait: Flags.duration({
+      char: 'w',
+      summary: messages.getMessage('flags.wait.summary'),
+      min: 0,
+      unit: 'minutes',
+      defaultValue: 0,
+    }),
+    'poll-interval': Flags.duration({
+      char: 'i',
+      summary: messages.getMessage('flags.poll-interval.summary'),
+      min: 15,
+      unit: 'seconds',
+      defaultValue: 30,
+    }),
+    name: Flags.string({
+      char: 'n',
+      summary: messages.getMessage('flags.name.summary'),
+      description: messages.getMessage('flags.name.description'),
+      parse: (name: string): Promise<string> => {
+        if (name.length > 10) {
+          throw messages.createError('error.SandboxNameLength', [name]);
+        }
+        return Promise.resolve(name);
+      },
+      exclusive: ['job-id'],
+    }),
+    'job-id': Flags.string({
+      char: 'i',
+      summary: messages.getMessage('flags.id.summary'),
+      description: messages.getMessage('flags.id.description'),
+      exclusive: ['name'],
+    }),
+    'use-most-recent': Flags.boolean({
+      char: 'l',
+      summary: messages.getMessage('flags.use-most-recent.summary'),
+      description: messages.getMessage('flags.use-most-recent.description'),
+    }),
+    'target-org': Flags.optionalOrg({
+      char: 'o',
+      summary: messages.getMessage('flags.targetOrg.summary'),
+      description: messages.getMessage('flags.targetOrg.description'),
+    }),
+  };
+  public static readonly state = 'beta';
+  protected readonly lifecycleEventNames = ['postorgcreate'];
```

Comment:
what does this do?  FYI, I moved this org hook stuff to core
https://github.com/forcedotcom/sfdx-core/pull/551/files#diff-4779a8d1c824c742a1c8bf52502a699d56d2d6d275d31c95146f1e2e650862a5R32